packages/forgetti/test/__snapshots__/hooks.test.ts.snap

```diff
@@ -53,6 +104,33 @@ function Example(props) {
 }"
 `;
 
+exports[`hooks > should optimize useMemo with 0 dependencies 1`] = `
+"import { useMemo as _useMemo } from \\"react\\";
+import { $$cache as _$$cache } from \\"forgetti/runtime\\";
+import { useMemo } from 'react';
+function Example(props) {
+  let _c = _$$cache(_useMemo, 2),
+    _v = _c[0] ||= [],
+    _v2 = _c[1] ||= (() => props.value())();
+  return _v2;
```

Comment:
IMHO the transformation could be further optimized:

```diff
- let _c = _$$cache(_useMemo, 2),
+ let _c = _$$cache(_useMemo, 1),
-    _v = _c[0] ||= [],
-    _v2 = _c[1] ||= (() => props.value())();
+    _v1 = _c[0] ||= (() => props.value())();
- return _v2;
+ return _v1;
```

Reply:
There's no "easy" way to know if the array would be empty. I'm just aggressively optimizing the `dependency` assuming that the value can be arbitrary (for example, `useMemo(() => example, myDeps)`

Reply:
I see, thanks!